docs/changelog/5_x.rst

```diff
@@ -0,0 +1,343 @@
+
+7) Since the state of "feature" is now a straightforward fast-forward w.r.t.
+   the "main" branch we can finally merge "feature" back into main in VCS and
+   then merge the EdgeDB branch as well (or rename "feature" EdgeDB branch
+   into "main", if the old branch is no longer needed).
+
+
+Update pgvector extension
+-------------------------
+
+A new HNSW (Hierarchical Navigable Small Worlds) index has been added to the ``pgvector`` extension. Just like IVFFlat indexes there are three flavors of HNSW corresponding to different operations:
+
+* ``ext::pgvector::hnsw_euclidean``
+* ``ext::pgvector::hnsw_ip``
+* ``ext::pgvector::hnsw_cosine``
+
+We have also updated the mechanism for tuning all of the indexes provided in this extension. The ``probes`` (for IVFFlat) and ``ef_search`` (for HNSW) parameters can now be accessed via ``ext::pgvector::Config`` object.
+
+The current config value can be found using this query:
+
+.. code-block:: edgeql-repl
+
+    db> select ext::pgvector::Config.probes;
+    {1}
+
+Updating the value can be done by ``configure session`` command:
+
+.. code-block:: edgeql-repl
+
+    db> configure session
+    ... set ext::pgvector::Config::probes := 5;
+    OK: CONFIGURE SESSION
+
+It is also possible to restore the default config value:
+
+.. code-block:: edgeql-repl
+
+    db> configure session reset ext::pgvector::Config::probes;
+    OK: CONFIGURE SESSION
+
+
+Authentication
+--------------
+
+We've expanded our ``auth`` extension with "WebAuthn" and "Magic Links" functionality.
+
+* Add WebAuthn authentication.
+  (:eql:gh:`#6791`, :eql:gh:`#6792`)
+
+* Add Magic Links passwordless provider.
+  (:eql:gh:`#6826`)
+
+* Update auth UI to support multiple email factor providers.
+  (:eql:gh:`#6874`)
+
+* Move app details configuration into ``AuthConfig``.
+  (:eql:gh:`#6754`)
+
+
+Additional changes
+==================
+
+Performance
+-----------
+
+We've made some changes to query caching aimed at improving the query recompilation in case of schema changes.
+
+We've also improved processing of large schemas and migrations.
+
+
+EdgeQL
+------
+
+* Allow omitting ``union`` in ``for`` if the body is a statement.
+  (:eql:gh:`#6810`)
+
+  If the ``for`` query body involves a statement such as ``insert``, ``update``, ``delete``, etc. you no longer need to write the ``union`` keyword and parenthesise the statement expression:
+
+  .. code-block:: edgeql-diff
+
+      for name int {'Alice', 'Billie', 'Cameron'}
+    - union (
+      insert User { name := name }
+    - )
+
+* Add ``administer vacuum()`` command.
+  (:eql:gh:`#6663`)
+
+  The command ``administer vacuum()`` can take a list of object types, multi properties, multi links or links with link properties. There is also named only argument ``full`` that controls how thorough the cleaning is going to be. All of the arguments can be omitted. In case no target types are specified everything accessible to the user will be vacuumed.
```

Comment:
```suggestion
  The command ``administer vacuum()`` can take a list of object types, multi properties, multi links or links with link properties. There is also a named only argument ``full`` that controls how thorough the cleaning will be. All of the arguments can be omitted. In case no target types are specified, everything accessible to the user will be vacuumed.
```